binders.tex

```diff
@@ -1,13 +1,15 @@
 \begin{syntax}
   binders ::= binder (, binder)* ;
   \
-  binder ::= type-expr variable-ident;
+  binder ::= type-name variable-ident;
```

Comment:
@davidcok this looks suspiciously like #62. Did somehow that branch get merged in this one?